src/components/content-tab-settings.tsx

```diff
@@ -17,7 +17,7 @@ interface ContentTabSettingsProps {
 
 function SettingsRow( { children, label }: PropsWithChildren< { label: string } > ) {
 	return (
-		<tr className="align-top">
+		<tr className="align-top" aria-label={ label }>
```

Comment:
This ARIA label addition results in announcing the same string twice — e.g, `Site name Site name`, which I believe is a disorienting experience for users relying upon assistive technologies. 

My perception is that we added this ARIA label purely for automated testing purposes. I suggest we solely query for the target PHP version or, if we feel it is notabley beneficial to also assert the version resides within a PHP version settings cell, we rely upon a test ID. My recommendation would be the former. 

<details><summary>Target PHP version only diff</summary>

```diff
diff --git a/src/components/content-tab-settings.tsx b/src/components/content-tab-settings.tsx
index 65b9630..c993480 100644
--- a/src/components/content-tab-settings.tsx
+++ b/src/components/content-tab-settings.tsx
@@ -17,7 +17,7 @@ interface ContentTabSettingsProps {
 
 function SettingsRow( { children, label }: PropsWithChildren< { label: string } > ) {
 	return (
-		<tr className="align-top" aria-label={ label }>
+		<tr className="align-top">
 			<th className="text-nowrap text-a8c-gray-50 pb-4 ltr:pr-6 rtl:pl-6 ltr:text-left rtl:text-right font-normal">
 				{ label }
 			</th>
diff --git a/src/components/tests/content-tab-settings.test.tsx b/src/components/tests/content-tab-settings.test.tsx
index 6174079..745eec8 100644
--- a/src/components/tests/content-tab-settings.test.tsx
+++ b/src/components/tests/content-tab-settings.test.tsx
@@ -191,7 +191,7 @@ describe( 'ContentTabSettings', () => {
 
 			( useGetPhpVersion as jest.Mock ).mockReturnValue( '8.2' );
 			rerender( <ContentTabSettings selectedSite={ selectedSite } /> );
-			expect( within( screen.getByLabelText( 'PHP Version' ) ).getByText( '8.2' ) ).toBeVisible();
+			expect( screen.getByText( '8.2' ) ).toBeVisible();
 		} );
 
 		it( 'changes PHP version and restarts site when site is running', async () => {
@@ -231,7 +231,7 @@ describe( 'ContentTabSettings', () => {
 
 			( useGetPhpVersion as jest.Mock ).mockReturnValue( '8.2' );
 			rerender( <ContentTabSettings selectedSite={ selectedSite } /> );
-			expect( within( screen.getByLabelText( 'PHP Version' ) ).getByText( '8.2' ) ).toBeVisible();
+			expect( screen.getByText( '8.2' ) ).toBeVisible();
 		} );
 	} );
 } );

```

</details>

Reply:
> This ARIA label addition results in announcing the same string twice — e.g, `Site name Site name` — which I believe is a disorienting experience for users relying upon assistive technologies.
>
> My perception is that we added this ARIA label purely for automated testing purposes. I suggest we solely query for the target PHP version or, if we feel it is notabley beneficial to also assert the version resides within a PHP version settings cell, we rely upon a test ID. My recommendation would be the former.

Good catch @dcalhoun 🙇 ! Yeah, I added the label only for testing purposes. The idea was to provide a way to access the container where the PHP version is displayed, to avoid false positives when querying across the UI a generic text like `8.2`. But I overlooked the undesired side effect it has in accessibility. Probably, my concern about matching other texts will be uncommon, so I lean towards the solution you shared about simply querying that text.

Reply:
I've applied this suggestion in https://github.com/Automattic/studio/pull/233/commits/7f07de72b564c32ddf0f240238e83728c72a7c7a and https://github.com/Automattic/studio/pull/233/commits/8ea4db0361eebb102ee24f20069cef4b219789bc.